core/src/main/java/co/cask/wrangler/steps/transformation/Reverse.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright © 2016-2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.wrangler.steps.transformation;
+
+import co.cask.wrangler.api.*;
+
+import java.util.List;
+
+@Usage(
+        directive = "reverse",
+        usage = "reverse <column>",
+        description = "Reverses a string so that it prints backwards"
+)
+public class Reverse extends AbstractStep {
+    // Columns of the column to be upper-cased
+    private String col;
```

Comment:
The code style is wrong. We use two spaces instead of four. Please reformat using the intellij profile.